src/components/InviteOptionsModal.tsx

```diff
@@ -63,13 +63,15 @@ const InviteOptionsModal = ({ recipient, onClose }: Props) => {
     </SafeAreaView>
   )
 }
+const { height, width } = Dimensions.get('window')
 
 const styles = StyleSheet.create({
   container: {
-    height: variables.height,
-    width: variables.width,
+    height,
+    width,
     flex: 1,
```

Comment:
`flexGrow: 1` and and inline styles on the the touchable seem to fix the display issues for me!
```
<Touchable onPress={handleClose} borderless={true} hitSlop={variables.iconHitslop} style={{ paddingHorizontal: variables.contentPadding }}>
  <Times />
</Touchable>
```

Reply:
@kathaypacific please disregard this has the same issue as seen in the screenshot on the top of the modal.

Reply:
i think that the problem you observe is with the Send amount screen, instead of with this modal. it's not expected that a modal should cover / infringe on the status bar, however you can see in the screenshot that the send amount screen header is encroaching, which is why it's peeping through. i'll see if i can fix that header instead.

![Screenshot_20220913-152546_Alfajores (dev)](https://user-images.githubusercontent.com/20150449/189913524-28768e32-eb59-434b-a5f3-b4da8b819350.jpg)

Reply:
@MuckT i'll merge this PR now and create a follow up to address the header issue since it is not strictly related to this modal. i started making some changes to the header but it was not a trivial change since it affects a few screens so a separate PR would be easier to review anyway :)